mirdata/salami.py

```diff
@@ -27,14 +27,15 @@
 
 import mirdata.utils as utils
 import mirdata.download_utils as download_utils
+import mirdata.jams_utils as jams_utils
 
 INDEX = utils.load_json_index('salami_index.json')
 METADATA = None
 DATASET_DIR = 'Salami'
 ANNOTATIONS_REMOTE = download_utils.RemoteFileMetadata(
-    filename='salami-data-public-master.zip',
-    url='https://github.com/DDMAL/salami-data-public/archive/master.zip',
-    checksum='f88b3455f1f2443458d094f603c0c1c7',
+    filename='salami-data-public-hierarchy-corrections.zip',
```

Comment:
Since we're updating this, we should update the info at the top of the module, to be clear which annotation version we're using!